.github/workflows/ios_deploy_firebase.yml

```diff
@@ -0,0 +1,64 @@
+# TODO Fix this workflow
+
+name: iOS - Deploy production build to Firebase
+
+on:
+  push:
+    branches:
+      - develop
+
+jobs:
+  build_and_deploy_ios:
+    name: Build & Deploy iOS
+    runs-on: macOS-latest
+    environment: staging
+    env:
+      TEAM_ID: ${{ secrets.TEAM_ID }}
+      MATCH_PASSWORD: ${{ secrets.MATCH_PASSWORD }}
+      KEYCHAIN_PASSWORD: ${{ secrets.KEYCHAIN_PASSWORD }}
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+
+      - name: Install SSH key
+        uses: webfactory/ssh-agent@v0.4.1
+        with:
+          ssh-private-key: ${{ secrets.SSH_PRIVATE_KEY }}
+
+      - name: Setup Flutter environment
+        uses: subosito/flutter-action@v2
+        with:
+          channel: 'stable'
+          flutter-version: '3.3.10'
+
+      - name: Get flutter dependencies
+        run: flutter pub get
+
+      - name: Run code generator.
+        run: flutter packages pub run build_runner build --delete-conflicting-outputs
+
+      - name: Setup .env
+        env:
+          ENV_PRODUCTION: ${{ secrets.ENV_PRODUCTION }}
+        run: |
+          echo $ENV_PRODUCTION > .env
+
+      - name: Set up Ruby
+        uses: ruby/setup-ruby@v1
+        with:
+          ruby-version: '3.0'
+          bundler-cache: true
+          working-directory: 'ios'
+
+      - name: Install iOS dependencies
+        run: cd ./ios && bundle install && bundle exec pod install
+
+      - name: Match Adhoc
+        run: cd ./ios && bundle exec fastlane sync_adhoc_signing
+
+      - name: Deploy to Firebase App Distribution
+        run: cd ./ios && bundle exec fastlane build_and_upload_to_firebase
+        env:
+          FIREBASE_IOS_APP_ID: ${{ secrets.FIREBASE_IOS_APP_ID_PRODUCTION }}
+          FIREBASE_TOKEN: ${{ secrets.FIREBASE_TOKEN }}
+          BUILD_NUMBER:  {{ $GITHUB_RUN_NUMBER }}
```

Comment:
@markgravity Currently we can't get the iOS deployment to work 🥲 We have decided to revisit this in the future so that we can focus on the Flutter development for now 🙏🏻 

[Slack convo](https://nimble-co.slack.com/archives/C04Q38BNSPL/p1679645899982609?thread_ts=1679638550.985039&cid=C04Q38BNSPL)